components/blitz/src/omero/gateway/facility/ROIFacility.java

```diff
@@ -33,6 +33,7 @@
 import java.util.concurrent.ExecutionException;
 
 import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.lang.NotImplementedException;
```

Comment:
NB: Use of this seems new to me. Not that it's a bad idea, but we may need to make sure it's use is documented, etc.